api/src/main/java/ai/djl/util/Float16Utils.java

```diff
@@ -124,4 +125,24 @@ public static float halfToFloat(short half) {
         }
         return Float.intBitsToFloat((half & 0x8000) << 16 | (exp | mant) << 13);
     }
+
+    /**
+     * Converts a byte array into a float array.
+     *
+     * @param bytes the byte array.
+     * @param order the byte order to use.
+     * @return a float array.
+     */
+    public static float[] fromByteArray(byte[] bytes, ByteOrder order) {
```

Comment:
I'm actually not sure if this really belongs to here,

return the value between -1.0f - 1.0f seems not general enough, it might only work for specific audio data processing use case.

Maybe we should keep this in AudioUtils.java

Reply:
I can move it to SampledAudioFactory